### ISSUE TITLE
Headers outdated?

### ISSUE THREAD

**Issue by LogicalSoIutions** [bjhiltbrand/drone_mobile_python#11]
Can't login at all :/

**Comment by LogicalSoIutions** [comment]
https://github.com/Hacksore/drone-mobile - has working in JS if you care to update

**Comment by bjhiltbrand** [comment]
Hmm, that's odd. I just tried re-loading the integration, as well as manually logging in using Postman calls. Both worked as expected. Looking at Hacksore's code, his authentication stuff is MUCH cleaner than mine, however. I will need to look at porting it to python and using it to make authentication more robust. As far as this issue goes, I'm not quite sure what's up with your implementation. Can you add the following section to your configuration.yaml file, restart HA and then try to re-load the plugin?

`logger:
  default: error
  logs:
    # log level for a custom component
    custom_components.drone_mobile: debug
    drone_mobile: debug`
    
  This should allow my debugging messages to display in the home assistant logs and give me an idea of where this is failing for you. Please let me know if you have any quesitons. Thank you.

**Comment by LogicalSoIutions** [comment]
Whoops, just realized my first comment on the issue didn't have my error lol

```
Traceback (most recent call last):
  File "demo.py", line 17, in <module>
    print(r.status()) # Print the status of all vehicles
AttributeError: 'Vehicle' object has no attribute 'status'
```

I'm a bit confused - what configuration.yaml file?

**Comment by bjhiltbrand** [comment]
Oh, I think I see. Are you just looking at this package and trying to use the demo script? I wrote this with the intent to use it with Home Assistant (There is a companion integration for Home Assistant that uses this code to make the calls to the DroneMobile servers). That is where I referenced the configuration.yaml from. That's the configuration file that Home Assistant uses and it's how you would set logging for that system.

I now see that I think you are just working with this library as a standalone. The demo script was committed before everything in the underlaying framework was changed around, and I forgot to update the demo script to take advantage of the new framework. I apologize for that. I can work on re-writing the demo script and update it.
